component/badge.css

```diff
@@ -0,0 +1,123 @@
+#badge-component{
+    display: flex; 
+    flex-direction: column;
+ }
+ 
+ 
+ 
+ #badge-component #badge-onicon{
+     
+ }
+ 
+ .badge{
+     display: inline-block;
+     position: relative;
+     margin: 1rem 4rem;
+ }
+ 
+ .badge .number{
+     display: flex;
+     justify-content: center;
+     align-items: center;
```

Comment:
as this is repeating, you can make separate css class for it and add in html class
for ex. d-flex, d-center etc.

Reply:
ok